.dockerignore

```diff
@@ -1,3 +1,4 @@
 bin
 .certs
 .tmp
+.vscode
```

Comment:
😅  https://github.com/moby/buildkit/commit/22c87dd8f53e2359f5e7faaf4f7c2453612c1f3e

Reply:
sorry missed this. will revert from both .gitignore and .dockerignore